include/openmc/math_functions.h

```diff
@@ -161,6 +161,40 @@ extern "C" double maxwell_spectrum(double T);
 
 extern "C" double watt_spectrum(double a, double b);
 
+//==============================================================================
+//! Samples an energy from the Gaussian energy-dependent fission distribution.
+//!
+//! Samples from a Normal distribution with a given mean and standard deviation
+//! The PDF is defined as s(x) = (1/2*sigma*sqrt(2) * e-((mu-x)/2*sigma)^2
+//! Its sampled according to 
+//! http://www-pdg.lbl.gov/2009/reviews/rpp2009-rev-monte-carlo-techniques.pdf
+//! section 33.4.4
+//!
+//! @param mean mean of the Gaussian distribution
+//! @param std_dev standard deviation of the Gaussian distribution
+//! @result The sampled outgoing energy
+//==============================================================================
+
+extern "C" double normal_variate(double mean, double std_dev);
+
+//==============================================================================
+//! Samples an energy from the Muir (Gaussian) energy-dependent distribution.
+//!
+//! This is another form of the Gaussian distribution but with more easily 
+//! https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-05411-MS
+//!  
+//! modifyable parameters
```

Comment:
Formatting got a little weird here.

Reply:
keen to fix this but it is not clear what needs changing

Reply:
Change it from
```C++
//! This is another form of the Gaussian distribution but with more easily 
//! https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-05411-MS
//!  
//! modifyable parameters
//!
//! @param e0 peak neutron energy [eV]
```
to
```C++
//! This is another form of the Gaussian distribution but with more easily
//! modifyable parameters
//! https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-05411-MS
//!  
//! @param e0 peak neutron energy [eV]
```

Reply:
I would also use "modifiable"

Reply:
Yeah, thats pretty clear I couldnt see the issue :)